QModManager/HarmonyPatches/DisableDevErrorReporting.cs

```diff
@@ -3,13 +3,17 @@
     using HarmonyLib;
     using UnityEngine;
 
-    [HarmonyPatch(typeof(SentrySdk), nameof(SentrySdk.Start))]
+    [HarmonyPatch]
     internal static class SentrySdk_Start_Patch
     {
         // This patch destroys any SentrySdk object
         // The SentrySdk class is the class that sends errors logged with Debug.LogError or Debug.LogException to the Subnautica developers
 
         [HarmonyPrefix]
+        [HarmonyPatch(typeof(SentrySdk), nameof(SentrySdk.Start))]
+#if !SUBNAUTICA_STABLE
+        [HarmonyPatch(typeof(SentrySdk), nameof(SentrySdk.TrySendingEvent))]
```

Comment:
So on all configs _except_ `SUBNAUTICA_STABLE` we are patching _both_ `Start` _and_ `TrySendingEvent` methods?

Reply:
Ahh yes adding the second one was because of the patch timing change and I left the original because well... I forgot it

Reply:
Actually removed the trysend as I moved the patch time back before that start so it will work again.